vllm/entrypoints/openai/protocol.py

```diff
@@ -16,6 +19,8 @@
 from vllm.transformers_utils.tokenizer import AnyTokenizer
 from vllm.utils import random_uuid
 
+# yapf: enable
```

Comment:
Don't think we need to disable yapf here.

Reply:
I did have to actually - there was an issue with `yapf` and something else (`isort` maybe?) conflicting with how imports were ordered or structured actually - @mgoin helped me debug it a week or two ago.

Reply:
I see. Could you move `yapf` upwards so only the minimum number of lines have to be excluded?

Reply:
Iirc this might be but I’ll double-check

Reply:
Finished double-checking this and it turns out that some of the refactors between when this was added and now (including moving some things to `chat_utils`) resulted in this no longer being necessary, so they have been removed. 

d5b169b4